packages/devtools_app/lib/src/shared/new_issue.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../devtools.dart' as devtools;
+import 'globals.dart';
+import 'utils.dart';
+
+const newDevToolsIssueUriDisplay = 'github.com/flutter/devtools/issues/new';
+
+Uri newDevToolsIssueUri({String? issueDetails}) {
+  final issueBodyItems = _issueLinkDetails();
+  if (issueDetails != null) issueBodyItems.add(issueDetails);
```

Comment:
nit: I would pre-pend the other details instead of appending at the end. That way in the github issue, the exception is first and the less pertinent devtools version information is at the bottom